README.md

```diff
@@ -78,6 +86,37 @@ pip install -U --pre mxnet -f https://dist.mxnet.io/python/cu100mkl
 ```
 
 There are multiple versions of MXNet pre-built package available. Please refer to [mxnet packages](https://gluon-crash-course.mxnet.io/mxnet_packages.html) if you need more details about MXNet versions.
+</details>
+
+<details>
+<summary>
+
+## PyTorch
+</summary>
+
+GluonCV supports Python 3.5 or later. The easiest way to install is via pip.
+
+## Stable Release
+The following commands install the stable version of GluonCV and PyTorch:
+
+```bash
+pip install gluoncv --upgrade
+pip install torch==1.6.0 torchvision==0.7.0 # for cuda 10.2
+```
+There are multiple versions of PyTorch pre-built package available. Please refer to [PyTorch](https://pytorch.org/get-started/previous-versions/) if you need other versions.
+
+
+**The latest stable version of GluonCV is 0.8 and we recommend PyTorch 1.6.0**
+
+## Nightly Release
+
+You may get access to latest features and bug fixes with the following commands which install the nightly build of GluonCV:
+
+```bash
+pip install gluoncv --pre --upgrade
+pip install torch==1.6.0 torchvision==0.7.0 # for cuda 10.2
```

Comment:
not sure if this will install cuda version?? for conda it will detect gpu, but for pip I don't think so

Reply:
This will install the cuda 10.2 version as shown in pytorch website.
![image](https://user-images.githubusercontent.com/16021460/100150990-5b12f900-2e55-11eb-817b-329d465f3308.png)

Reply:
shall we use +cu102 version by default in the installation guide?

Reply:
At least for pytorch, I think so. 10.2 is the default for both pytorch 1.6 and 1.7.